web-app/js/portal/filter/ui/FilterGroupPanel.js

```diff
@@ -138,7 +138,7 @@ Portal.filter.ui.FilterGroupPanel = Ext.extend(Ext.Container, {
 
         var panelOrder = [
             Portal.filter.GeometryFilter,
-            Portal.filter.DateFilter,
```

Comment:
Any reason why we need to retain `DateFilter`, along with `WmsDateFilter`?  Can it not just be rolled in to the same class, if it's always the same behaviour that we need?